sormas-backend/src/test/java/de/symeda/sormas/backend/sormastosormas/SormasToSormasFacadeTest.java

```diff
@@ -157,58 +162,80 @@ protected SormasToSormasShareInfo createShareInfo(
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getServerAccessDataFileName()).thenReturn(DEFAULT_SERVER_ACCESS_DATA_CSV);
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystoreName()).thenReturn("default.sormas2sormas.keystore.p12");
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystorePass()).thenReturn("certPass");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getId()).thenReturn(DEFAULT_SERVER_ID);
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystoreName()).thenReturn("sormas_a.sormas2sormas.keystore.p12");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystorePass()).thenReturn("1234");
 		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getTruststoreName()).thenReturn("sormas2sormas.truststore.p12");
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getTruststorePass()).thenReturn("truster");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getTruststorePass()).thenReturn("password");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getRootCaAlias()).thenReturn("S2SCA");
 	}
 
 	protected void mockSecondServerAccess() {
 		File file = new File("src/test/java/de/symeda/sormas/backend/sormastosormas/serveraccesssecond");
 
 		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getPath()).thenReturn(file.getAbsolutePath());
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getServerAccessDataFileName()).thenReturn(SECOND_SERVER_ACCESS_DATA_CSV);
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystoreName()).thenReturn("second.sormas2sormas.keystore.p12");
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystorePass()).thenReturn("certiPass");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getId()).thenReturn(SECOND_SERVER_ID);
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystoreName()).thenReturn("sormas_b.sormas2sormas.keystore.p12");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getKeystorePass()).thenReturn("1234");
 		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getTruststoreName()).thenReturn("sormas2sormas.truststore.p12");
-		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getTruststorePass()).thenReturn("trusteR");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getTruststorePass()).thenReturn("password");
+		Mockito.when(MockSormasToSormasConfigProducer.sormasToSormasConfig.getRootCaAlias()).thenReturn("S2SCA");
 	}
 
 	@Specializes
-	private static class MockSormasToSormasConfigProducer extends SormasToSormasConfigProducer {
+	public static class MockSormasToSormasConfigProducer extends SormasToSormasConfigProducer {
```

Comment:
Get rid of this and the SormasToSormasConfigProducer

Reply:
produces removed